client/pages/_app.tsx

```diff
@@ -16,12 +17,19 @@ function App({ Component, pageProps }: AppProps) {
   }
 
   return (
-    <QueryClientProvider client={queryClientRef.current}>
-      <AppProvider>
-        <DynamicNavBar />
-        <Component {...pageProps} />
-      </AppProvider>
-    </QueryClientProvider>
+    <div>
```

Comment:
maybe we can just use `React.Fragment` here?

Reply:
Changed.